mlrun/model_monitoring/helpers.py

```diff
@@ -245,29 +254,40 @@ def get_tsdb_connection_string(
     )
 
 
-def _get_tsdb_profile(
-    project: str = "",
-    secret_provider: typing.Optional[typing.Callable[[str], str]] = None,
+def _get_profile(
+    project: str,
+    secret_provider: Optional[Callable[[str], str]],
+    profile_name_key: str,
```

Comment:
I think this name could be more descriptive.
```suggestion
    secret_key: str,
```

Reply:
Honestly, it was my first variable name. I think that `secret_key` is too generic. Maybe `profile_name_secret_key`? A bit long but can work